### PR TITLE
Add spo page

### PR DESCRIPTION
**Pull request by guavajellyaaron**

### CODE REVIEW

**Review by happygaijin — APPROVED** [review]
LGTM. Just a couple of comments.

File: src/components/SpoPortal/Form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          SPO Portal
+        </h2>
+      </div>
+      <div className="col-span-12">
+        <div className="shadow-lg rounded-lg border-2 border-primary-100 p-4 md:p-8 mb-8">
+          <h3 className="text-white text-3xl font-bold text-center mb-4">
+            How it Works
+          </h3>
+          <p className="text-lg text-center text-dark-350">
+            Purchasing a Handle during for SPOs is a 4-step process, starting
```
[the remark this answers is not in the document]
Comment (guavajellyaaron) [reply src/components/SpoPortal/Form/index.tsx:98]: Nice Catches! I'm actually going to get more feedback on the copy.

File: src/components/SpoPortal/Form/index.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+              operators wallet for the purchase
+            </li>
+            <li>
+              If you make a payment using a wallet other than the wallet
+              attached to your stake pool, your payment will be refunded minus a
+              50 ADA processing fee
+            </li>
+          </ol>
+          <p>
+            Pricing for each Handle ranges from 10-500 $ADA, depending on the
```
[the remark this answers is not in the document]
Comment (guavajellyaaron) [reply src/components/SpoPortal/Form/index.tsx]: Copy has been fixed

**Comment by cjkoepke** [comment]
@guavajellyaaron Testing out the preview URL, a couple of immediate things:

1. This currently breaks the minting page (specifically: `TypeError: Cannot read properties of undefined (reading 'toLocaleString')`).
2. Let's change the title: SPO Portal to SPO Claiming Portal
3. When I tried to search anything, the `/search` endpoint returned a 502 error and there was no feedback in the UI as to what happened.

**Pull request merged**